blog/_posts/2023-09-11-10.2-change-collection.md

```diff
@@ -60,6 +69,99 @@ There's a [dedicated post](https://stormearthandlava.com/blog/2023/09/10/t31-fir
 
 # General Shaman Changes
 
+## Splintered Elements
+
+- {{ site.data.talent.splinter }} now grants you 20% Haste plus an additional 4% Haste per extra target hit by {{ site.data.talent.primordial_wave }}, rather than 10% Haste per target hit.
+
+> Source: [Blue Post](https://us.forums.blizzard.com/en/wow/t/guardians-of-the-dream-ptr-development-notes/1666552/6) (2023-09-27)
+
+<div class="card">
+  <div class="card-header" id="spe">
+    <div class="h2 mb-0">
+      <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#spe-body" aria-expanded="false" aria-controls="spe-body">
+        General notes
+      </button>
+    </div>
+  </div>
+  <div id="spe-body" class="collapse" aria-labelledby="spe">
+    <div class="card-body" markdown="1">
+
+- This change mostly buffs {{ site.data.talent.splinter }} in Single Target and nerfs it at 5 and more targets. This is overall a good change for builds that use Splintered as their single target was their main issue.
```

Comment:
This change primarily buffs {{ site.data.talent.splinter }} in Single Target and nerfs it at 5 or more targets. Builds using Splintered Elements usually struggle in Single Target, so this change helps them close the gap a bit.